### ISSUE TITLE
The heat tree for common species

### ISSUE THREAD

**Issue by taowis** [grunwaldlab/metacoder#318]
For the opposite of differential heat tree, could we print the heat tree with common species for both groups?

**Comment by zachary-foster** [comment]
Yes. Do you mean species that are not differentially abundant (high p value) or species that both have reads in two groups of samples? Both are possible. You would filter the taxa in the `taxmap` object before plotting.
